influxql/query_executor_test.go

```diff
@@ -143,7 +143,7 @@ func TestQueryExecutor_Abort(t *testing.T) {
 }
 
 func TestQueryExecutor_ShowQueries(t *testing.T) {
-	q, err := influxql.ParseQuery(`SELECT count(value) FROM cpu`)
+	_, err := influxql.ParseQuery(`SELECT count(value) FROM cpu`)
 	if err != nil {
 		t.Fatal(err)
 	}
```

Comment:
These first four lines can just be removed - they appeared to be a copy-paste error, and we have coverage for parsing this query elsewhere.